.changeset/smooth-walls-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nacelle/nacelle-js": patch
+---
+
+feat(breaking!): update transformLineItem functions & add tests
```

Comment:
I'm not sure if this is breaking.

Reply:
I wasn't sure.  I thought it might be because users would now get errors they hadn't before.  Happy to update though

Reply:
@NWRichmond - since the transform function now passes along _all_ the parameters supplied by users, it means if incorrect parameters are supplied, Shopify errors will be returned.  Before this update, we pruned only the accepted params and passed them along.  I was on the fence here between breaking and not breaking

Reply:
Ah okay, that makes sense. In that case, updating this changeset to describe that scenario would be helpful.